Ivy support

When compiling an app using this library with ionic 5 and angular 9, with ivy turned on, I get the following error during compilation

```
Compiling ionic-native-http-connection-backend : main as commonjs

ERROR in Failed to compile entry-point ionic-native-http-connection-backend (main as commonjs) due to compilation errors:
node_modules/ionic-native-http-connection-backend/dist/native-http-module.ts:16:22 - error NG1010: Value at position 0 in the NgModule.imports of NativeHttpModule is not a reference: [object Object]

16             imports: [HttpClientModule],
                        ~~~~~~~~~~~~~~~~~~

[ERROR] An error occurred while running subprocess ng.
```

Turning off ivy in my app's tsconfig.json fixes this however

--- a/package.json
+++ b/package.json
@@ -5,10 +5,11 @@
   "main": "dist/index.js",
   "typings": "dist/index.d.ts",
   "scripts": {
+    "build": "rimraf aot dist && ngc",
     "commit": "git-cz",
-    "test": "jest",
+    "postinstall": "ngcc",
     "semantic-release": "semantic-release",
-    "build": "rimraf aot dist && ngc"
+    "test": "jest"
   },
   "repository": {
     "type": "git",
@@ -33,6 +34,7 @@
     "@angular/core": "^9.0.1",
     "@angular/forms": "^9.0.1",
     "@angular/platform-browser": "^9.0.1",
+    "@angular/router": "^9.0.1",
     "@commitlint/cli": "^8.3.5",
     "@commitlint/config-conventional": "^8.3.4",
     "@ionic-native/core": "^5.21.5",
@@ -58,6 +60,7 @@
     "zone.js": "^0.10.2"
   },
   "peerDependencies": {
+    "angular/core": "^9.0.0",
     "@ionic-native/http": "^5.12.0"
   },
   "config": {
@@ -117,8 +120,5 @@
         "message": "chore(release): ${nextRelease.version} [skip ci]\n\n${nextRelease.notes}"
       }
     ]
-  },
-  "dependencies": {
-    "@angular/router": "^9.0.1"
   }
 }

